<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMain</class>
 <widget class="QMainWindow" name="WindowMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Reflections</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot_main" native="true">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>500</width>
      <height>465</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_wave_type">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>25</y>
      <width>115</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>PP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_wave_type">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>25</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wave type</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_layers">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>75</y>
      <width>350</width>
      <height>130</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>130</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Layers parameters</string>
    </property>
    <widget class="QLabel" name="label_vp_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;g/sm&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_vp1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_vp">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vp</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_dn2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>100</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>3.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_dn1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_layer_2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>25</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Layer 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vs">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>75</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vp_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>75</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vp_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_layer_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>25</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Layer 1</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_vs2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>75</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_density">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>100</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Density</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_vp2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>50</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>4000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_vs1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>75</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>1500</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_visual">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>295</y>
      <width>350</width>
      <height>130</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>130</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Visual parameters</string>
    </property>
    <widget class="QGroupBox" name="groupBox_parts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Part to draw</string>
     </property>
     <widget class="QCheckBox" name="checkBox_abs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>absolute</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_imag">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>imaginary</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_real">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>real</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_bound">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Boundaries</string>
     </property>
     <widget class="QLabel" name="label_angles">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>50</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Angles</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_r">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>75</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_bound_min">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>min</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_angle_max">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>50</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>89</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>85</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_angle_min">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>89</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_r_min">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>75</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <double>-5.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>-1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_bound_resize">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>25</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>auto</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_r_max">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>75</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_angle">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>50</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_r">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>75</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_bound_max">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>25</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>max</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_approx">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Approximation</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_calc">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>350</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Calculation</string>
    </property>
    <widget class="QCheckBox" name="checkBox_recalc">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Recalculate automatically</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_compute">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>23</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_r0">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;R&lt;/span&gt;&lt;span style=&quot; font-size:12pt; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_r">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>25</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_g">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;G&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_g">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_export">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Export</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QPushButton" name="pushButton_graph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_window">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>25</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save window</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_table">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>25</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to table</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_approx">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
